components/fxa-client/src/fxa_client.udl

```diff
@@ -189,13 +189,12 @@ interface FirefoxAccount {
   ///       - This parameter is used for metrics purposes, to identify the
   ///         UX entrypoint from which the user triggered the signin request.
   ///         For example, the application toolbar, on the onboarding flow.
+  ///   - `service` - list of services to request.
   ///   - `metrics` - optionally, additional metrics tracking parameters.
   ///       - These will be included as query parameters in the resulting URL.
   ///
   [Throws=FxaError]
-  string begin_oauth_flow([ByRef] sequence<string> scopes, [ByRef] string entrypoint);
-  
-
+  string begin_oauth_flow([ByRef] sequence<string> scopes, [ByRef] string entrypoint, [ByRef] sequence<string> service);
```

Comment:
We can use default values for the udl so that the iOS tests don't fail if that's a preference: https://mozilla.github.io/uniffi-rs/latest/proc_macro/functions.html#default-values

Alternatively, we can also fix the breaking tests too. :)

Reply:
Yeah, sorry I related too late that I shared the wrong doc link. 🤦